Dockerfile.builder

```diff
@@ -55,3 +56,9 @@ RUN apt-get update \
 RUN curl -sfL https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -xzC /usr/local && \
         GO111MODULE=on go get github.com/gordonklaus/ineffassign@1003c8bd00dc2869cb5ca5282e6ce33834fed514 && \
         go clean -cache -modcache
+
+#
+# Install Helm
+#
+RUN curl -sfL https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz | tar -xzC /tmp && \
```

Comment:
Why are we installing helm in cilium-builder image? This does not seem to be required.